js/src/main/scala/Main.scala

```diff
@@ -133,7 +133,7 @@ object Main {
           implicit val extrCtx: Opt[typer.ExtrCtx] = N
 
           val vars: Map[Str, typer.SimpleType] = Map.empty
-          val tpd = typer.typeTypingUnit(tu, allowPure = true)(ctx.nest, raise, vars)
```

Comment:
The parameter name was changed. We should keep setting it to `true`.

Reply:
Thanks. It's fixed.